src/containers/flows/MintDropNFT/steps/Finished/Finished.tsx

```diff
@@ -11,7 +11,7 @@ type FinishedProps = {
 const Finished = (props: FinishedProps) => {
 	return (
 		<section className={styles.Container}>
-			<span>Your Pets were minted successfully!</span>
+			<span>Your Beasts were minted successfully!</span>
```

Comment:
Is it possible to change this label based on how many were minted?

1: `Your Beast was minted successfully!`
2+: `Your Beasts were minted successfully!`